CONTRIBUTING.md

```diff
@@ -1,6 +1,8 @@
 # Contributing Guidelines
 
-This is the contributing guidelines for the client library.
+This document outlines the contributing guidelines for the Provena client library. Contributions are welcome for new features, bug fixes, and documentation updates.
+
+---
 
 ## Usage
```

Comment:
actually, is the usage section here meant to be in the main readme? The intro says this has contributing guidelines but usage guidelines is the first thing and is not flagged in the intro. A reader would expect to first see contribution guidelines as opposed to usage guidelines in this doc given whta the intro says. And if someone is reading a contributing.md i would say its safe to assume they are familiar with usage or atlaest with the usage documentation?

My suggestion is to move usage into the main readme before testing section.

Reply:
resolved in call.